SpywareFilter/sections/mobile_allowlist.txt

```diff
@@ -15,10 +15,6 @@
 @@||google-analytics.com^$app=jp.co.alc.eow
 @@||tr.line.me^$app=jp.co.alc.eow
 @@||tr.outbrain.com^$app=jp.co.alc.eow
-! https://github.com/AdguardTeam/AdguardFilters/issues/81513
-@@||google-analytics.com/analytics.js$script,domain=m.dcinside.com|mupload.dcinside.com
-@@||ssl.google-analytics.com/ga.js$script,domain=m.dcinside.com|mupload.dcinside.com
-@@||dcinside.com$cookie=_ga
```

Comment:
Remove last line maybe enforce bake cookie on every refresh instead one of year (if `_ga` is still removed globally).

Reply:
fixed